### ISSUE TITLE
Address test issues

### ISSUE THREAD

**Issue by kentcdodds** [codesandbox/codesandbox-client#3580]
This one was hard to write in an issue because inconsistencies, so I made a video: https://youtu.be/89QocluKVUs

Here are some of the codesandboxes:
- https://codesandbox.io/s/dazzling-dan-eschs
- https://codesandbox.io/s/jovial-firefly-zx2mj
- https://codesandbox.io/s/github/kentcdodds/advanced-react-patterns/tree/create-workshop-app

Let me know if you have questions.

**Comment by CompuIves** [comment]
First of all, apologies for the delay on this. I think we now have a fix, could you verify whether it works?

**Comment by kentcdodds** [comment]
Hooray! Thank you very much @CompuIves! It works perfectly now!!

https://codesandbox.io/s/github/kentcdodds/react-testing-library-examples

Thank you thank you!

**Comment by kentcdodds** [comment]
The next big thing for me would be `jest.mock` which is tracked here: https://github.com/codesandbox/codesandbox-client/issues/513 so I won't bother making a new issue.

**Comment by petermarcoen** [comment]
Hi @kentcdodds,

I am getting `jest.useRealTimers is not a function` on all tests in https://codesandbox.io/s/github/kentcdodds/react-testing-library-examples

Same issue I have been having with my own CodeSandbox, came here through this issue: https://github.com/codesandbox/codesandbox-client/issues/513

**Comment by kentcdodds** [comment]
I'm giving up on the in-browser jest stuff and I've moved my stuff to the node template. It's much slower, but at least it'll work now. Though it looks like codesandbox isn't picking up my latest pushes at https://github.com/kentcdodds/react-testing-library-examples

Hopefully it'll resolve itself soon.

**Comment by CompuIves** [comment]
Hmm, yes, Jest is indeed not fully supported in the browser. We have implemented the basics of Jest by porting it to the browser, but some things (like mocks and fake timers) are harder to port, so that's currently not implemented. Like Kent said, it's better to use a node template in case you want all features of Jest working 😄 .

> Though it looks like codesandbox isn't picking up my latest pushes at https://github.com/kentcdodds/react-testing-library-examples

I took a look at this, we've been hitting some rate limits with GitHub which prevents us to update. This week we've deployed several fixes to this, so it should be much more reliable now again with updating!
